app/services/peoplesoft_service.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
```

Comment:
This is largely a refactor of https://github.com/sul-dlss/hydra_etd/blob/main/app/services/registrar_data_importer.rb with the populate/update methods being handled in etds_controller now and this focusing on reader and registrar actions.